server/apps/immich/src/api-v1/user/dto/create-user.dto.ts

```diff
@@ -1,8 +1,8 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsNotEmpty, IsOptional } from 'class-validator';
+import { IsNotEmpty, IsEmail } from 'class-validator';
 
 export class CreateUserDto {
-  @IsNotEmpty()
+  @IsEmail()
```

Comment:
Should we also include `IsNotEmpty` decorator since this is the required property?

Reply:
I've added unit tests for checking undefined / invalid / valid, so `IsEmail` already checks that is set and not empty